src/Tizen.NUI/src/public/GraphicsTypeManager.cs

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright(c) 2019 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+namespace Tizen.NUI
+{
+
+    /// <summary>
+    /// GraphicsTypeManager class to manage various types.
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public class GraphicsTypeManager
+    {
+
+        /// <summary>
+        /// Creates private GraphicsTypeManager object.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        private GraphicsTypeManager()
+        {
+            _typeConverter = new GraphicsTypeConverter();
+        }
+
+        /// <summary>
+        /// </summary>
+        /// <returns>Returns Singleton instance of GraphicsTypeManager</returns>
+        /// <since_tizen> 6 </since_tizen>
+        public static GraphicsTypeManager Instance
```

Comment:
Only methods can have a `<returns>` section, please change it to `<value>` section instead.